duckduckgo/duckduckgo-privacy-extension
# Hotfix - Top blocked not showing up in FF

**Reviewer:** @jdorweiler 



## Description:
A last minute change during the previous release resulted in the top blocked stats being always hidden in FF.
This PR fixes that.
Since it's a hotfix I also bumped the manifest pin so we can release to FF immediately.


## Steps to test this PR:
1. build and reload FF extension
2. initially no top blocked line is shown 
3. after 30 page loads top blocked should show up

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
